variables.tf

```diff
@@ -138,7 +138,13 @@ variable "opentelemetry" {
         cpu    = string,
         memory = string,
       }),
-    })
+    }),
+    tolerations = list(object({
```

Comment:
https://www.terraform.io/language/expressions/type-constraints#experimental-optional-object-type-attributes pls (and `value` in there also)